app/src/components/GrantList.vue

```diff
@@ -23,12 +23,20 @@ import GrantCard from 'src/components/GrantCard.vue';
 import useCartStore from 'src/store/cart';
 // --- Types ---
 import { FilterNavButton, FilterNavItem, Grant, GrantMetadataResolution } from '@dgrants/types';
+// hard-coded grant ID List
+import grantIDList from 'src/CustomGrantIndexList.json';
```

Comment:
I believe we said we'd fetch this file from a URL to avoid having to redeploy the app every time we want to change the list. So it'd be hosted on e.g. github in a repo (TBD what repo) and do something like this. We could probably just keep it in this repo for now

```typescript
const jsonFetch = (url: string) => fetch(url).then((res: any) => res.json());
const url = 'https://raw.githubusercontent.com/dcgtc/dgrants/top-level-grant-filter/app/src/CustomGrantIndexList.json'
const grantIdList = await jsonFetch(url);
```